frontend/src/employee-frontend/components/document-templates/template-editor/TemplateContentEditor.tsx

```diff
@@ -470,7 +470,7 @@ const BasicsEditor = React.memo(function BasicsEditor({
           hideErrorsBeforeTouched
           data-qa="process-definition-number"
         />
-        {processDefinitionNumber.value().trim().length > 0 && (
+        {processDefinitionNumber.state.trim().length > 0 && (
```

Comment:
tarttis olla kyllä joku linter sääntö, että näitä ei lipsahda koodiin

Reply:
Linter-sääntö ei ehkä toimi koska on hyvin kontekstisidonnaista kumpaa kuuluu käyttää. Enemmän ehkä API-design ongelma. Mutta tää on kyllä yksi yleisimpiä runtime virheiden aiheuttajia.